internal/controller/config/config.go

```diff
@@ -46,7 +46,7 @@ type Config struct {
 	JobPrefix                string        `json:"job-prefix"               validate:"required"`
 	MaxInFlight              int           `json:"max-in-flight"            validate:"min=0"`
 	Namespace                string        `json:"namespace"                validate:"required"`
-	Tags                     stringSlice   `json:"tags"                     validate:"min=1"`
+	Tags                     stringSlice   `json:"tags"                     validate:"min=0"`
```

Comment:
May as well drop the `validate` tag entirely.

```suggestion
	Tags                     stringSlice   `json:"tags"`
```